### PR TITLE
CI working group meeting notes 2021-11-10

### PR DESCRIPTION
**Pull request by jfpanisset**
Signed-off-by: Jean-Francois Panisset <panisset@gmail.com>

### CODE REVIEW

File: meetings/2021-11-10.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+
+## Apologies
+
+## New items
+
+* Conan builds in aswf-docker!
+
+    * [https://github.com/AcademySoftwareFoundation/aswf-docker/pull/136](https://github.com/AcademySoftwareFoundation/aswf-docker/pull/136) and follow on PRs
+
+        * All the builds go through a release workflows, had some issues with GLU and GLFW, got resolved by adding the Conan Central repository to get header only packages
```
Comment (aloysbaillet) [inline meetings/2021-11-10.md:32]: glew had an issue, not glu

File: meetings/2021-11-10.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+    * Test suite tool recommendations
+
+    * Conan support?
+
+* Larger GHA build instances (request from OpenVDB)
+
+    * Some can be addressed by adjusting build matrix, but may still be useful for large single builds
+
+        * Andrew: GitHub library functionality is now available, two things we can do, composable actions, which is "macro-ing" Actions together, "an Action of Actions", as well as "public workflows", available not just to your organization, but also publicly.
+
```
Comment (aloysbaillet) [inline meetings/2021-11-10.md:109]: Wrong links?

**Review by aloysbaillet — DISMISSED** [review]
Great notes as usual! Thanks @jfpanisset !

File: meetings/2021-11-10.md (hunk 1 above)
Comment (jfpanisset) [reply meetings/2021-11-10.md]: Right, GLU is long gone...

**Review by aloysbaillet — APPROVED** [review]

File: meetings/2021-11-10.md (hunk 2 above)
Comment (jfpanisset) [reply meetings/2021-11-10.md:109]: Good catch, links posted by Andrew in Zoom are long gone, but I think I've found corresponding / relevant links (copy-paste between Zoom and Google Docs is always a bit iffy).

**Pull request merged**